client/state/directly/reducer.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * External dependencies
+ */
+import { combineReducers } from 'redux';
+
+/**
+ * Internal dependencies
+ */
+import {
+	DIRECTLY_INITIALIZE
+} from 'state/action-types';
+import { createReducer } from 'state/utils';
+
+export const isInitialized = createReducer( false, {
+	[ DIRECTLY_INITIALIZE ]: () => true,
+} );
```

Comment:
@aduth should chime in here since he's the wizard behind it.

I personally don't use it and think we should find a burden of need before using it because it does introduce hidden overhead. It's very good for when we need memoized reducers for performance-sake, but that's rarer than it seems.

we must remember that every single reducer runs on every single Redux action dispatch…